packages/connection-manager/__tests__/shared/connectionManagerAgentLogic.ts

```diff
@@ -81,6 +81,19 @@ export default (testContext: { getAgent: () => ConfiguredAgent; setup: () => Pro
       expect(result.length).toBeGreaterThan(0)
     })
 
+    it('should get parties by filter', async () => {
```

Comment:
Can we also have some testcases, which only have one property (like only name), but more importantly have a test that shouldn't return anything.

Reply:
added additional test cases